### PR TITLE
Adding Inferentia AI/ML Module to Workshop

### PR DESCRIPTION
**Pull request by bkgardiner**
#### What this PR does / why we need it:
Adding the AIML Module to the work shop with Inferentia lab

#### Which issue(s) this PR fixes:

Fixes #

#### Quality checks

- [x] My content adheres to the style guidelines
- [x] I ran `make test` or `make e2e-test` and it was successful

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: website/docs/aiml/inferentia/inference.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+```
+
+The inference pod should be scheduled on the node provisioned by Karpenter. Check if the Pod is in it's ready state:
+
+:::note
+It can take up to 8 minutes to provision the node, add it to the EKS cluster, and start the pod.
+:::
+
+```bash timeout=360
+$ kubectl -n aiml wait --for=condition=Ready --timeout=8m pod/inference
```
Comment (niallthomson) [inline website/docs/aiml/inferentia/inference.md:81]: After the pod is running can you use `kubectl get node` to show the `aws.amazon.com/neuron` resources on the node? I think that would help tie in the node to the pod request a bit better.

File: website/docs/aiml/inferentia/_category_.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "collapsed": false
```
Comment (niallthomson) [inline website/docs/aiml/inferentia/_category_.json:2]: This shouldn't be expanded by default, set this to `true`

**Review by niallthomson — CHANGES_REQUESTED** [review]
Hey @bkgardiner looks like a pretty solid PR. Left a couple of minor comments.

**Comment by bkgardiner** [comment]
Updated the pull request to resolve your comments.

**Review by niallthomson — DISMISSED** [review]

**Pull request merged**